### ISSUE TITLE
VaTimeInput bugs

### ISSUE THREAD

**Issue by smellyshovel** [epicmaxco/vuestic-ui#2768]
# 1

Latest version.

```
            <va-time-input
              v-if="useIndexProxy"
              v-model="rawSearchRequestEntry.timeTo"
              :disabled="!rawSearchRequestEntry.dateTo"
              manual-input
              autocomplete="off"
              icon=""
            />
```

When disabled, the component still allows to modify the input manually (when the `manual-input` prop is on):

https://user-images.githubusercontent.com/9782236/205644333-5ef5b4c7-5f74-42a6-8bcb-2aa82099d178.mp4

Maybe there's the same issue with date-input, didn't check.

# 2

Less of a bug, but still.

When `icon=""` is set, the icon is indeed not shown. Though it's still possible to target it with keyboard navigation.
Would be nice to have some way to disable the icon completely.

**Comment by smellyshovel** [comment]
@aluarius I meant that the calendar/time-dropdown doesn't open indeed, but it's still possible to enter the date/time manually using the input field. When the component is `disabled`, that shouldn't be the case.

**Comment by aluarius** [comment]
Yes, i've understand, but can't repro it anyway -> in case `disabled` and `manual-input` props are `true`, i'm still not able to focus input and change anything inside it.

Anyway, as was said, i have added additional restriction.

Safari/Chome?

**Comment by aluarius** [comment]
Discussion moved to slack.
